src/solver/conv/conv_ocl_dir2Dfwd.cpp

```diff
@@ -41,6 +41,16 @@ using ProblemDescription = miopen::conv::ProblemDescription;
 bool ConvOclDirectFwd::IsApplicable(const ExecutionContext& ctx,
                                     const ProblemDescription& problem) const
 {
+#ifdef WORKAROUND_SWDEV_503936
```

Comment:
```suggestion
#if WORKAROUND_SWDEV_503936
```
#ifdef is always evaluated to TRUE, even if WORKAROUND_SWDEV_503936 is 0